charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml

```
@@ -274,6 +277,29 @@ data:
       - target_label: instance
         replacement: kube-state-metrics
       metric_relabel_configs:
+      # We want to drop all metrics that
+      # do not belong to the shoot
+
+      # Add a temp label to all ksm metrics
+      - target_label: __tmp_keep
+        replacement: false
```

Can you quote the temp values to avoid confusion with bools?
```suggestion
        replacement: "false"
```